### DISCUSSION TITLE
Build web-based example

### DISCUSSION THREAD

**Discussion by ivanperez-keera** [ivanperez-keera/dunai#363] [Ideas]
Create example with a running compiler for web (GHCJS, wasm, etc.).

The state of web & haskell is unclear. It would be nice if it was built using a compiler that is still maintained.

**Comment by turion** [comment]
That's a fantastic idea! Don't you have a version of haskanoid in a browser running since years?

**Comment by ivanperez-keera** [comment]
Yeap, but I'm not sure it compiles at the moment. GHCJS's API had breaking changes multiple times and it's poorly documented.

**Comment by ivanperez-keera** [comment]
(Not to critizice the author(s), it's a very big undertaking and generally just very hard.)

**Comment by geroldmeisinger** [comment]
There is this company "Keera Studio" (maybe you've heard of them? ;) ) who did a web example a while ago: https://keera.co.uk/posts/2020/05/28/building-a-reactive-calculator-in-haskell-1-5/

**Comment by ivanperez-keera** [comment]
Haha. Yeap, rings a bell lol
